site/en/workflows.md

```diff
@@ -1376,29 +1569,10 @@ If your computer is not that powerful you may try this workflow. It works fine f
     </tr>
     <tr>
       <td>7</td>
-      <td>ocrd-tesserocr-segment-region</td>
-      <td></td>
-    </tr>
-    <tr>
-    <td>7a</td>
-      <td>ocrd-segment-repair</td>
-      <td>-P plausibilize true</td>
-    </tr>
-    <tr>
-      <td>9</td>
-      <td>ocrd-tesserocr-deskew</td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>10</td>
-      <td>ocrd-cis-ocropy-clip</td>
+      <td>ocrd-tesserocr-segment</td>
       <td></td>
```

Comment:
Also helpful here, because you have dewarping afterwards, so better polygonalize the textline coords.

```suggestion
      <td>-P shrink_polygons true</td>
```